### PR TITLE
Feature/nuc123 vxp listview component

### PR DESCRIPTION
**Pull request by ercang**
image
image
image
image
image

### CODE REVIEW

File: docs/.vuepress/components/VxpListViewDoc.vue [hunk 1]
```diff
@@ -0,0 +1,257 @@
...
+      <Label text="VxpListView Options" class="demo-panel__title"/>
+
+     <FlexboxLayout flexWrap="wrap">
+        <Label text="List Height" class="option-label"/>
+        <TextField id="vxplistview_listheight_textfield" v-model="listViewHeight" class="option-input"/>
+      </FlexboxLayout>
+
+      <FlexboxLayout flexWrap="wrap">
+        <Label text="Is list reversed?" class="option-label"/>
+        <input id="vxplistview_reversed_cb" v-model="isListReversed" type="checkbox">
```
Comment (ahmetceylan) [inline docs/.vuepress/components/VxpListViewDoc.vue]: we can use vxpcheckbox

File: src/components/VxpListView.vue [hunk 2]
```diff
@@ -0,0 +1,280 @@
...
+  }
+
+  &::-webkit-scrollbar-thumb:vertical,
+  &::-webkit-scrollbar-thumb:horizontal {
+    height: 30px;
+    background-color: #ccc;
+  }
+
+  &::-webkit-scrollbar {
+    width: 8px;
```
Comment (ahmetceylan) [inline src/components/VxpListView.vue]: please use unit function

File: docs/.vuepress/components/VxpListViewDoc.vue (hunk 1 above)
Comment (mylmz10) [reply docs/.vuepress/components/VxpListViewDoc.vue]: and VxpLabel

File: src/components/VxpListView.vue (hunk 2 above)
Comment (ercang) [reply src/components/VxpListView.vue]: fixed!

File: docs/.vuepress/components/VxpListViewDoc.vue (hunk 1 above)
Comment (ercang) [reply docs/.vuepress/components/VxpListViewDoc.vue]: fixed

File: docs/.vuepress/components/VxpListViewDoc.vue [hunk 3]
```diff
@@ -0,0 +1,257 @@
+<template>
+  <StackLayout>
+    <StackLayout class="demo-panel">
+      <Label text="VxpListView Options" class="demo-panel__title"/>
+
+     <FlexboxLayout flexWrap="wrap">
+        <Label text="List Height" class="option-label"/>
+        <TextField id="vxplistview_listheight_textfield" v-model="listViewHeight" class="option-input"/>
```
Comment (mylmz10) [inline docs/.vuepress/components/VxpListViewDoc.vue]: We have to import TextField

Comment (ercang) [reply docs/.vuepress/components/VxpListViewDoc.vue]: its already imported.

**Pull request merged**